source/differentiation/sd/symengine_tensor_operations.cc

```diff
@@ -46,7 +46,7 @@ namespace Differentiation
         {
           std::string out;
           for (unsigned int i = 0; i < rank; ++i)
-            out += dealii::Utilities::to_string(indices[i]);
+            out += dealii::std::to_string(indices[i]);
```

Comment:
Serves me right for using sed and not having symengine installed. Let me fix this.